assets/src/scripts/components/dailyValueHydrograph/selectors/legend-data.spec.js

```diff
@@ -0,0 +1,175 @@
+import {select, selectAll} from 'd3-selection';
+import {drawSimpleLegend, drawTimeSeriesLegend} from '../legend';
+import {lineMarker, rectangleMarker, textOnlyMarker} from '../../../d3-rendering/markers';
+import {getLegendMarkerRows} from './legend-data';
+import {configureStore} from '../../../store';
+
+
+describe('DV: Legend module', () => {
+
+    const TEST_STATE = {
+        observationsData: {
+            timeSeries: {
+                '12345': {
+                    type: 'Feature',
+                    id: '12345',
+                    properties: {
+                        phenomenonTimeStart: '2018-01-02',
+                        phenomenonTimeEnd: '2018-01-05',
+                        timeStep: ['2018-01-02', '2018-01-03', '2018-01-04', '2018-01-05'],
+                        result: ['5.0', '4.0', '6.1', '3.2'],
+                        approvals: [['Approved'], ['Approved'], ['Approved'], ['Estimated']],
+                        nilReason: [null, 'AA', null, null],
+                        qualifiers: [null, null, ['ICE'], ['ICE']],
+                        grades: [['50'], ['50'], ['60'], ['60']]
+                    }
+                }
+            }
+        },
+        observationsState: {
+            currentTimeSeriesId: '12345',
+            showSeries: {
```

Comment:
This should be removed. The observationsState object does not include a showSeries property.

Reply:
This still needs to be removed.